### DISCUSSION TITLE
King of Fighters 2001 Plus

### DISCUSSION THREAD

**Discussion by Deathsoul56** [mamedev/discussions#44] [General]
For a very long time I use MAMEP 0.145 + MAMEPGUI, among the games I play frequently is the KOF saga, one of the roms I played was King of Fighters 2001 plus (kf2k1pls), now update to MAME 0.254 and the only rom that does not work is kf2k1pls which is not listed in the available section.
Is there a way to add it?

**Comment by rb6502** [comment]
That game appears to have been removed at some point, I don't know why.

**Reply by cuavas** [reply]
I don’t think MAME itself has ever supported a `kf2k1pls` set.  It might be in HBMAME or something.
